### PR TITLE
Finish creating layout of the Cycle page

### LINKED ISSUE #10: Create the layout of the Cycle Page
Create the layout of the cycle page with correct proportions and fill them with filler texts.
You don't need to implement any of the functionalities

- [x] Create four boxes for Current Cycle, Average period length, Average cycle length, and Period History
- [x] Make the page scrollable
Screen Shot 2022-11-06 at 2 27 32 PM

### PR DESCRIPTION
**Pull request by Manpreet-Bhatti**
- Created boxes to lay out the blueprint of the Cycle page (with text and placeholder images)
- Added scrolling ability to layout wrapper (needs a deeper look -- don't know if scrolling was done correctly)

Closes #10

### CODE REVIEW

**Comment by Manpreet-Bhatti** [comment]
As a side comment: we should probably consider putting `gradle.build` in the `.gitignore`. Thoughts?

File: app/src/main/java/com/example/theperiodpurse/ui/cycle/CycleScreen.kt [hunk 1]
```diff
@@ -1,4 +1,134 @@
...
+        super.onCreate(savedInstanceState)
+        setContent {
+            ThePeriodPurseTheme {
+                Surface(color = MaterialTheme.colors.background) {
+                    CycleScreenLayout()
+                }
+            }
+        }
+    }
+}
```
Comment (leowrites) [inline app/src/main/java/com/example/theperiodpurse/ui/cycle/CycleScreen.kt]: We should only have one entry to our App which is in MainActivity, I don't think it's necessary to have this class here.

File: app/src/main/java/com/example/theperiodpurse/ui/cycle/CycleScreen.kt [hunk 2]
```diff
@@ -1,4 +1,134 @@
...
+
+@Composable
+fun CurrentCycleBox() {
```
Comment (leowrites) [inline app/src/main/java/com/example/theperiodpurse/ui/cycle/CycleScreen.kt]: Something I see the tutorial emphasize on is passing in `modifier: Modifier = modifier` to make the component reusable. Let's follow that convention.

File: app/src/main/java/com/example/theperiodpurse/ui/cycle/CycleScreen.kt [hunk 3]
```diff
@@ -1,4 +1,134 @@
...
+    Card(
+        Modifier
+            .fillMaxWidth()
+            .height(300.dp),
+        elevation = 2.dp, shape = RoundedCornerShape(5)
+    ) {
+        Text(
+            text = "Current cycle",
```
Comment (leowrites) [inline app/src/main/java/com/example/theperiodpurse/ui/cycle/CycleScreen.kt]: We should move the text to `string.xml` and use `getString(R.string.string_name)` here. Let's do this for all the other texts.

**Review by leowrites — COMMENTED** [review]
Overall, great work! I just left some minor comments and should be some simple fixes.

**Comment by leowrites** [comment]
I think scrolling using .verticalScroll() is fine

> As a side comment: we should probably consider putting `gradle.build` in the `.gitignore`. Thoughts?

Hmmm gradle.build contains all the dependencies right?

**Comment by Manpreet-Bhatti** [comment]
> I think scrolling using .verticalScroll() is fine
> 
> > As a side comment: we should probably consider putting `gradle.build` in the `.gitignore`. Thoughts?
> 
> Hmmm gradle.build contains all the dependencies right?

I believe it's an overarching config file for your Android Studio project. Not a breaking change to ignore it but not bad to have it either 😅 just comes up in `git status`.

**Review by leowrites — APPROVED** [review]
Great work 👍

**Review by elenawzy — APPROVED** [review]
looks good !!

**Pull request merged**